Makefile

```diff
@@ -155,6 +157,43 @@ clean-cov: ## Remove coverage report
 test: clean-cov manifests generate fmt vet envtest ## Run tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./... -coverprofile cover.out -v -timeout 0
 
+.PHONY: namespace
+namespace: ## Creates a namespace where to deploy Kuadrant Operator
+	kubectl create namespace $(KUADRANT_NAMESPACE)
+
+.PHONY: local-setup
+local-setup: export IMG := $(IMAGE_TAG_BASE):dev
+local-setup: ## Deploy locally kuadrant operator from the current code
```

Comment:
I like the `make local-setup`, it is kind of standard in kuadrant projects. 

This target replaces the `kind-create-kuadrant-cluster` make target in `make/kind.mk` file. It should be deleted.